### PR TITLE
grpc-httpjson-transcoding@0.0.0-20230607-ff41eb3

### PR DESCRIPTION
**Pull request by mmorel-35**
Used by:
* https://github.com/grpc-ecosystem/proto-field-extraction
* https://github.com/envoyproxy/envoy

Related to https://github.com/grpc-ecosystem/grpc-httpjson-transcoding/pull/97

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (grpc-httpjson-transcoding) have been updated in this PR. Please review the changes.

File: modules/grpc-httpjson-transcoding/0.0.0-20230607-ff41eb3/patches/grpc-httpjson-transcoding.patch [hunk 1]
```diff
@@ -0,0 +1,94 @@
+diff --git a/WORKSPACE b/WORKSPACE
+index 692ab12..c0d7f34 100644
+--- a/WORKSPACE
++++ b/WORKSPACE
```
Comment (fmeum) [inline modules/grpc-httpjson-transcoding/0.0.0-20230607-ff41eb3/patches/grpc-httpjson-transcoding.patch:4]: WORKSPACE should be ignored, do we need to patch it?

File: modules/grpc-httpjson-transcoding/0.0.0-20230607-ff41eb3/patches/grpc-httpjson-transcoding.patch [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+ 
+diff --git a/repositories.bzl b/repositories.bzl
+index 68d98be..56ee95e 100644
+--- a/repositories.bzl
++++ b/repositories.bzl
+@@ -15,7 +15,6 @@
+ ################################################################################
+ #
+ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+-load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
```
Comment (fmeum) [inline modules/grpc-httpjson-transcoding/0.0.0-20230607-ff41eb3/patches/grpc-httpjson-transcoding.patch:79]: Do we need this part of the patch?

**Review by fmeum — DISMISSED** [review]

File: modules/grpc-httpjson-transcoding/0.0.0-20230607-ff41eb3/presubmit.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  bazel:
+  - 7.x
+  - 6.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_flags:
+    - '--cxxopt=-std=c++17'
```
Comment (fmeum) [inline modules/grpc-httpjson-transcoding/0.0.0-20230607-ff41eb3/presubmit.yml:17]: `/std:c++17` is the correct flag on Windows with MSVC. You could split this task into one for Windows and one for the rest of the matrix.

**Review by fmeum — APPROVED** [review]

**Pull request merged**